test/integration/fury-test.es6

```diff
@@ -108,6 +109,17 @@ describe('Parser', () => {
       });
     });
 
+    it('should parse when returning element instances', (done) => {
+      // Modify the parse method to return an element instance
+      fury.adapters[fury.adapters.length - 1].parse = ({source}, cb) =>
```

Comment:
We could. This is actually an adapter that we define a few lines above specifically for the test, I just wanted to modify it in one test. I'd rather hold off on another dependency until we need it for more tests.